beetsplug/filetote.py

```diff
@@ -333,9 +335,6 @@ def process_artifacts(self, source_files, mapping):
             util.mkdirall(dest_file)
             dest_file = util.bytestring_path(dest_file)
 
-            # TODO: detect if beets was called with 'move' and override config
```

Comment:
This was a [legacy `TODO`](https://github.com/adammillerio/beets-copyartifacts/blob/master/beetsplug/copyartifacts.py#L165-L166) that is now resolved.

The `beets.config` module provides the _current_ config when referenced and not just the values in `config.yml`. Fixing _when_ the `self.operation` is obtained deprecates this TODO.